bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -211,7 +216,30 @@
 		<description>Is triggered 10 times in 5 seconds when the sensor detects a break-in attempt.</description>
 		<event>
 			<options>
-				<option value="ALARM">alarm</option>
+				<option value="ALARM">Alarm</option>
+			</options>
+		</event>
+	</channel-type>
+
+	<channel-type id="protectionPlusEvent">
+		<kind>trigger</kind>
+		<label>Protection Plus Event</label>
+		<description>Triggered when a Protection Plus capable device detects a break-in attempt.</description>
+		<event>
+			<options>
+				<option value="ALARM">Alarm</option>
+			</options>
+		</event>
+	</channel-type>
+
+	<channel-type id="vacationModeToggleEvent">
+		<kind>trigger</kind>
```

Comment:
Is this really an event or can it be implemented as a state? Does the handle really send a "no change" (0x0) message for the vacation mode or just 0x1 and 0x2?

Reply:
According to the specification this can only be implemented as an event: This event will only be emitted when the physical vacation mode button is pressed on the handle, it will not be sent when the vacation mode is set via a configuration message. Also (again according to the specs) the event is not repeated but only sent once.

On this screenshot from #6658 you can see that the handle indeed sends "No change" if the button was not pressed: https://user-images.githubusercontent.com/17730574/127765547-6209aa03-1b4e-48b5-b37d-14c60cf685d8.png